app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java

```diff
@@ -145,9 +145,9 @@ public void onError(Throwable e) {
                     public void onNext(Page<Client> clientPage) {
                         getMvpView().hideProgress();
                         if (clientPage.getPageItems().size() != 0) {
-                            long clientId = clientPage.getPageItems().get(0).getId();
-                            getMvpView().showClient(clientId);
-                            preferencesHelper.setClientId(clientId);
+                            Client clientCurrent = clientPage.getPageItems().get(0);
+                            getMvpView().showClient(clientCurrent);
```

Comment:
No, It's not. We are building the end user banking app. We need fresh data at every step.